### ISSUE TITLE
TTS output voice speed too fast

### ISSUE THREAD

**Issue by cantabile-kwok** [espnet/espnet#5238]
**Describe the bug**
When I use the pretrained model and official code to synthesize the TTS samples, I found that the output voice speed very fast. In FastSpeech2, it only becomes normal if I set duration alpha to be 1.5.

**Basic environments:**
 - OS information: "CentOS Linux 7 (Core)"
 - python version: Python 3.9.6 (default, Aug 18 2021, 19:38:01)  [GCC 7.5.0] :: Anaconda, Inc. on linux
 - espnet version: 202304
 - pytorch version: 1.11.0+cu102

**Task information:**
 - Task: TTS
 - Recipe: LibriTTS
 - ESPnet2

**To Reproduce**
```python
from espnet2.bin.tts_inference import Text2Speech
import kaldiio
import torchaudio


tts = Text2Speech.from_pretrained("pretrained/tts_train_xvector_conformer_fastspeech2_transformer_teacher_raw_phn_tacotron_g2p_en_no_space_train.loss.ave.zip", vocoder_file="pretrained_hifigan/checkpoint-2500000steps.pkl")

spemb_scp = kaldiio.load_scp("dump/xvector/eval1/spk_xvector.scp")
spemb = spemb_scp['260_123286']

wav = tts("This is generated using a pretrained vocoder from a checkpoint", spembs=spemb)['wav']
torchaudio.save("sample.wav", wav.unsqueeze(0), tts.fs)
```
This issue exists no matter I use the Fastspeech2 or tacotron checkpoint. The filepath above refers to the pretrained fastspeech2 on `espnet_model_zoo`. For the tacotron model, I have been using "pretrained/tts_train_xvector_transformer_raw_phn_tacotron_g2p_en_no_space_train.loss.ave.zip".

**Comment by cantabile-kwok** [comment]
Here are some Tacotron samples that I have generated for reference:
https://drive.google.com/file/d/1r0Z6yPd92So1dmiERemFRLUD17haEaFZ/view?usp=sharing

**Comment by iamanigeeit** [comment]
@cantabile-kwok I followed your steps but don't have this problem, here is my
[sample](https://github.com/espnet/espnet/files/11882150/sample.txt) (rename it to .wav)

I am on Ubuntu 20, Conda Python 3.9.13, GCC 9, PyTorch 1.11. Have you tried updating ESPnet?

**Comment by kan-bayashi** [comment]
Is the feature extraction setting of the vocoder the same as that of the text2mel model?
Please paste both model's configs.

**Comment by cantabile-kwok** [comment]
Thanks everybody! I have figured out the problem, which was because of a reason I have never thought of. The code snippet in the beginning of this issue does not produce any problems, but when I substitute the xvector into my own, the voice speed is indeed fast. It was my mistake to think that the original code works wrong.
To be exact, I found that the output duration is highly dependent on the extracted xvectors. I have compared my extraction script to the espnet official script and found we used a different mfcc config file (different frame length and shift to be exact). Although I have checked that the range of values stays the same with official ones, it somehow produces an xvector that makes the TTS model accelerate its voice speed. After I switched to the correct mfcc config, the voice speed issue is gone.
